tests/pfcwd/test_pfc_config.py

```diff
@@ -186,6 +191,7 @@ def stop_pfcwd(duthosts, rand_one_dut_hostname):
     Returns:
         None
     """
+    yield
```

Comment:
@antonptashnik Just wonder why add 'yield' here? `stop_pfcwd` fixture is use to stop pfcwd before test no after.
Could you please give a classification? thanks.

Reply:
the intention is not to leave PFCWD enabled after these tests. If it's disabled on test start only, then the last test will leave it enabled because tests enable it during execution.
Since PFCWD is disabled by default, I can disable PFCWD at the end of each test instead of at the beginning, so neither any test begins with it enabled not it's left enabled after all tests finish